CAIPs/caip-ipld_anchor_event.md

```diff
@@ -0,0 +1,150 @@
+---
+caip: <to be assigned>
+title: IPLD Anchor Events and Proofs 
```

Comment:
```suggestion
title: DAG timestamp proof
```

I think this title is a bit more clear. It's a proof that a particular DAG under a CID (or set of CIDs) were timestamped at a particular point in time.

Reply:
The summary and abstract needs to be updated to use this new language.

Reply:
dont think we should remove all anchor wording, but i do like the timestamp framing here as i tried to do the same in the details as well

Reply:
Agreed. Main thing here is just that people will understand what a timestamp is but we need to explain what we mean by anchors.

Reply:
@zachferland did we land on **IPLD timestamp proof**?

Reply:
yup!